XRTK.Lumin/Packages/com.xrtk.lumin/Runtime/Providers/InputSystem/Controllers/LuminController.cs

```diff
@@ -16,7 +16,7 @@
 
 #endif
 
-namespace XRTK.Lumin.Providers.Controllers
+namespace XRTK.Lumin.Providers.InputSystem.Controllers
```

Comment:
I didn't really feel the need to bury these into another empty folder. 

Is it okay that we just don't keep `InputSystem`? Bc technically the controllers should be able to work without them.